src/DynamoCore/Interfaces/IPreferences.cs

```diff
@@ -22,6 +22,8 @@ public interface IPreferences
         List<string> RecentFiles { get; set; }
         List<string> BackupFiles { get; set; }
         List<string> PackageDirectoriesToUninstall { get; set; }
+        List<string> CustomNodeFolders { get; set; }
+        List<string> PackageFolders { get; set; } 
```

Comment:
`XmlSerializer` requires it to be of type `List` so that the serializer knows what type to create the list from during file loading.